consensus/polybft/blockchain_wrapper.go

```diff
@@ -83,8 +82,7 @@ func (p *blockchainWrapper) CommitBlock(block *types.FullBlock) error {
 }
 
 // ProcessBlock builds a final block from given 'block' on top of 'parent'
-func (p *blockchainWrapper) ProcessBlock(parent *types.Header, block *types.Block,
-	callback func(*state.Transition) error) (*types.FullBlock, error) {
```

Comment:
Glad we got rid of this callback :raised_hands: 

I remember it was essential to have it validated after a block was executed since validators transitions were calculated by querying `ChildValidatorSet`, but now when tracking validators transitions is on the client side, it can be removed.